src/main/webapp/i18n/de/exam.json

```diff
@@ -458,6 +458,31 @@
             "alreadySubmitted": "Du hast bereits abgegeben.",
             "submissionNotInTime": "Du hast die Klausur nicht rechtzeitig abgegeben. Sie wird nicht bewertet!",
             "handInFailed": "Die Abgabe der Klausur ist fehlgeschlagen. Bitte versuche es erneut!"
+        },
+        "examStatus": {
+            "columnTitle": "Klausurstatus",
+            "preparation": {
+                "examPreparation": "Klausurvorbereitung",
+                "configureExercises": "Konfiguriere Klausuraufgaben: {{ amount }} Aufgabengruppen",
+                "registerStudents": "Registriere Studierende: {{ registered }} registiert",
+                "generateStudentExams": "Generiere Klausuren: {{ generated }} von {{ total }}",
+                "prepareExerciseStart": "Bereite Aufgabenstart vor"
+            },
+            "conduction": {
+                "examConduction": "Klausurdurchführung",
+                "startedExams": "Begonnene Klausuren",
+                "submittedExams": "Abgegebene Klausuren"
+            },
+            "correction": {
+                "examCorrection": "Klausurkorrektur",
+                "correctionRoundProgress": "Korrekturfortschritt in Runde {{ index }}",
+                "examReview": "Klausureinsicht",
+                "unset": "Nicht angesetzt",
```

Comment:
```suggestion
                "unset": "Nicht gesetzt",
```
I assume this translation is used when some value of the exam is not yet set to a proper value.

Reply:
The translation is used in order to indicate that the exam review phase is not set by the instructor. It might be personal preference but I think in this case "angesetzt" fits a bit better than "gesetzt". What do you think about this?
![image](https://user-images.githubusercontent.com/80622272/152125278-dfcd0da0-6b6e-4dd6-be42-52dc99039a13.png)

Reply:
Then I assumed the wrong context. In this case the current translation definitely makes more sense.